jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/EnumReader.java

```diff
@@ -28,6 +28,21 @@ public EnumReader(Class<?> enumType,
         _byName = byName;
     }
 
+    /**
+     * Detect whether a class is an enum or a subclass of an enum - e.g. an anonymous inner class
+     * inside an enum
+     * @param type the type to inspect
+     * @return <code>true</code> if effectively an enum
+     */
+    public static boolean isEnum(Class<?> type) {
```

Comment:
Why are these methods here, although used in different place? Probably belongs in caller, or a util-class of some type.

Reply:
Moved to a new utility class